Geometry/CMSCommonData/data/extend/cmsextent.xml

```diff
@@ -13,15 +13,20 @@
 	</ConstantsSection>
 	<SolidSection label="cmsextent.xml">
 		<Tubs name="CMStoZDC" rMin="[Rmin]" rMax="[CMSR2]" dz="56*m" startPhi="0*deg" deltaPhi="360*deg"/>
-		<Tubs name="ZDCtoFP420" rMin="[Rmin]" rMax="[CMSR2]" dz="120*m" startPhi="0*deg" deltaPhi="360*deg"/>
+		<Tubs name="ZDCtoVF200" rMin="[Rmin]" rMax="[CMSR2]" dz="26*m" startPhi="0*deg" deltaPhi="360*deg"/>
```

Comment:
@mundim - you must create a new version of this file in order not to change previous geometries. The rule for new files is as follows _name/year/version/name.xml_:
```
Geometry/CMSCommonData/data/cmsextent/2018/v1/cmsextent.xml
```